site/quickstart.md

```diff
@@ -0,0 +1,262 @@
+ 
+```bash
+imagetool cache addPatch --patchId 27342434 --version 12.2.1.3.0 --path /home/acmeuser/cache/p27342434_122130_Generic .zip -user username@mycompany.com --passwordEnv MYPWD
+```
+
+You need to provide the credential in the command line, it verifies the MD5 on the file system against the meta data on 
+Oracle Support for this patch number
+
+Then, you can run the command to create the image:
+
+```bash
+imagetool create --fromImage myosimg:latest --tag wls:12.2.1.3.0 --patches 27342434 --version 12.2.1.3.0 --useCache always
+```
+
+Sometimes, a WebLogic patch may require patching the OPatch binaries before applying them.  We recommend 
+downloading the latest OPatch's patch and setup the cache.  For example, the latest OPatch's patch is 28186730, 13.9
+.4.0.0.  You can use this command to setup the cache after downloading from Oracle Support.
+
+```bash
+imagetool cache addPatch --patchId 28186730 --version 13.9.4.0.0 --path /home/acmeuser/cache/p28186730_139400_Generic.zip
+```
+
+## Patching an existing image
+
+This use case allows you apply WebLogic patches to an existing image. It works similar to previous use cases, you 
+have the option to download automatically by the tool or manual downloading the patches yourself.
+
+Once the cache is setup, you can use the following command to update an image:
+
+```bash
+imagetool update --fromImage wls:12.2.1.3.0 --tag wls:12.2.1.3.4 --patches 27342434 --version 12.2.1.3.0 --useCache always
+```
+
+## Create an image with a WebLogic Domain using WebLogic Deploy Tool
+
+The image tool allows create a customized WebLogic domain using [WebLogic Deploy Tool](https://github.com/oracle/weblogic-deploy-tooling).  
+
+You can accomplish this by:
+
+Download the [WebLogic Deploy Tool from](https://github.com/oracle/weblogic-deploy-tooling/releases), then add it to the cache store
+
+```bash
+imagetool cache addInstaller --type wdt --version 2.2 --path /home/acmeuser/cache/weblogic-deploy.zip
+```
+
+Provide the command line options for WebLogic Deploy Tool
+
+```bash
+imagetool create --fromImage myosimg:latest --tag wls:12.2.1.3.0 --patches 27342434 --version 12.2.1.3.0 --useCache always --wdtVersion 2.2 --wdtArchive /home/acmeuser/wdt/domain1.zip --wdtDomainHome /u01/domains/simple_domain
+```
+
+The parameters mapping between Image Tool and WebLogic Deploy Tool are:
+
+| Image Tool         | WebLogic Deploy Tool |
+|--------------------|-------------------------|
+| --wdtArchive       | -archive_file           |
+| --wdtModel         | -model_file             |
+| --wdtVariables     | -variable_file          |
+| --run_rcu          | -run_rcu                |
+| --wdtDomainHome    | -domain_home            |
+
+
+The domain will be created under ```/u01/domains/base_domain``` if you do not specify ```--wdtDomainHome```.  
+
+Note: if you are creating a JRF domain and wants WebLogic Deploying Tool to create the rcu schemas for you, you can specify the connection info in the model [Specifying RCU 
```

Comment:
Note: if you are creating a JRF domain and wants WebLogic Deploying Tool to create the rcu schemas for you, ->
**Note:** If you are creating a JRF domain and want WebLogic Deploying Tool to create the RCU schemas for you,

connection info -> connection information